infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInternalRuntimeTest.java

```diff
@@ -137,6 +138,7 @@ public void setup() throws Exception {
         new OpenShiftInternalRuntime(
             13,
             5,
+            FAIL_ON_UNRECOVERABLE_EVENT,
```

Comment:
Can you add a case when the value is false?

Reply:
@garagatyi do you think it is ok to check one infra with `false` & other with `true`, or you think there should be both cases handled in both infras ?

Reply:
Since they might have their own bugs we should have tests for both